.github/workflows/conformance-aks-v1.11.yaml

```diff
@@ -148,7 +148,7 @@ jobs:
           elif [ "${{ github.event_name }}" = "schedule" ]; then
             curl https://api.github.com/repos/cilium/cilium/branches/v1.11 > branch.json
             SHA=$(jq -r '.commit.sha' branch.json)
-            OWNER=v1.11
+            OWNER=v111
```

Comment:
Because I got scared of any special character. If it can't handle dots, god knows what it can handle :shrug: 

Switched to `v1-x`. If it breaks, you'll owe me the fix :stuck_out_tongue_closed_eyes: